### ISSUE TITLE
O> 20180314 RChain Active Members Meeting 

### ISSUE THREAD

**Issue by Tonyprisca13** [rchain/bounties#469]
## ZOOM LINK FOR RAM MEETING 14 MAR: [https://zoom.us/j/197490909](https://zoom.us/j/197490909)
2018-March-14, 20:00 UTC (after Greg's Debrief)
Sydney +11, Beijing +8, Greece+2, Lagos +1, Amsterdam +1,
New York -5, Seattle -8

- [x] Prepare report for Greg's Debrief 

- [x] Prepare [RAM meeting agenda](https://docs.google.com/document/d/1jvUn8AKLmvnibNk2KT-tqJdWDV5FoYZfVPIF8czxy64/edit?ts=5aa05d54#)

- [x] Any key discussion points anyone wants to address should be added to the agenda and we can 
  allocate time accordingly

@Makys @Raob @dcplnau @ojimadu @kitblake @lapin7  happy for additional agenda items to be added by any of the members. Please add to [agenda](https://docs.google.com/document/d/1jvUn8AKLmvnibNk2KT-tqJdWDV5FoYZfVPIF8czxy64/edit?ts=5aa05d54#)

# Budget: $2000

**Comment by ResonanceXX** [comment]
Thanks @Tonyprisca13 The Zoom is always important.

**Comment by makys** [comment]
I added:
"New Member WG" update to the agenda
Update on the bot to sync between discord and Github to identify paid members to be paid bounty
Update on the bounty app @dckc

**Comment by Jake-Gillberg** [comment]
@Tonyprisca13 I'm confused about the $2,000 budget. Can you break it down for me?

**Comment by dckc** [comment]
I got some support for [my suggestion to use a label for the agenda](https://github.com/rchain/bounties/issues/381#issuecomment-367167794); I hope the approach can continue.

> leaving a trail of all relevant comments / discussions in the actual issue would be a great idea. Much better than having in scattered between issue, Discord channels and video call recordings -- @pmoorman 

> Let's decide about it in the meeting of 2018-02-28 and leave the label on until decision has been made. (good norm) -- @lapin7 in #391

**Comment by BelovedAquila** [comment]
Think we should address the bounty payments system, on how to better its efficiency in payments procedure. Because so far I think our bounty payments system is lagging behind @lapin7 and @Tonyprisca13

**Comment by Keaycee** [comment]
#483

**Comment by dckc** [comment]
Looks like I'm available to attend today after all.

**Comment by dckc** [comment]
regarding
> we should address the bounty payments system, on how to better its efficiency in payments procedure.

In today's meeting:

> HJ: there are now 3 of us working on invoices. ETA for Feb invoice drafts to be sent out is tomorrow.

**Comment by dckc** [comment]
@Tonyprisca13 is claiming half the reward for this issue; what's the basis of that?

**Comment by Tonyprisca13** [comment]
I create the agenda and Gregs debrief for 20140314 and also the Gregs debrief and scribe for 20140307

**Comment by dckc** [comment]
OK, that's perhaps 15 to 20% of the value of the meeting; not 50%. Please lower your reward vote.

**Comment by Tonyprisca13** [comment]
@dckc what is the basis of that?

**Comment by dckc** [comment]
Seriously? I'm starting to doubt that you are participating in good faith.

This is obvious, but I'll spell it out:

If there are 10 people at the meeting, splitting the reward evenly is a reasonable place to start. Someone who did extra work may merit 15, 20, or 25% of the reward. But if the meeting was actually an exchange of ideas among the participants and not just one person doing a presentation or something, clearly no one person merits any more reward than that.

**Comment by Tonyprisca13** [comment]
@dckc  i am participating in good faith, i voted what was justified

**Comment by Tonyprisca13** [comment]
Everyone one on this issue has the right to vote what they think is justified

**Comment by Tonyprisca13** [comment]
@dckc  i created an issue i vote what i think is just based on  it what is my offence

**Comment by jimscarver** [comment]
@dckc tony created the agenda and Gregs debrief for 20140314 and also the Gregs debrief and scribe for 20140307. Attending the meeting is worth no more than 1 or 2%. You have not answered how you earned 20%

**Comment by Tonyprisca13** [comment]
@dckc voted 20 percent for Jake Gilberg for a comment why is that if the highest should be 20 percent?

**Comment by dckc** [comment]
@jimscarver I was not asked how I earned 20%. But I infer that you're asking now, so I'll re-iterate what's already a matter of record: I demo'd the bounty app and served as scribe.

@9rb you were the guide for that meeting; I'm interested in your input.

I voted 20% for Jake for his input on #483 concerns surrounding translation work.

Perhaps attending silently is worth only 1 or 2%; but I was starting with the assumption that attendees are also participants. Splitting the reward among the participants seems like a reasonable place to start. The record shows 8 participants, which suggests 12.5%.

**Comment by Tonyprisca13** [comment]
@dckc  12.5% per participant has that been the standard for meeting issues? or you just feel like implementing it on this particular issue

**Comment by kitblake** [comment]
tl;dr I think the budget for these weekly RAM Meeting issues should be $800, and rewards should be more or less evenly distributed to those who contributed. 

Let's review the antecedents to this issue. Back in Jan/Feb we had Chantal organizing the weekly RAM meeting. Here's an example of the issue description:
```
# goal: Organize the meeting [RAM 20180124](https://docs.google.com/document/d/1jvUn8AKLmvnibNk2KT-tqJdWDV5FoYZfVPIF8czxy64/edit#heading=h.u4pck0fugly0)

- [x] set agenda before 2018-01-24
- [x] scan issues in Github and Discord to talk about
- [x] publish meeting announcement in media channels
- [x] participate in the meeting
- [x] make minutes and report
- [x] prepare report for Greg's Debrief 2018-01-31

# budget: $2000 
# time: 2 weeks
```
So Chantal would do all of the above, guide the RAM meeting, make a report, and then attend Greg's Debrief where she would make a live verbal report. 

At each RAM meeting people would present things. Maybe a demo of an ongoing project, or a report from a project group, or show a document in progress, etc. These presentations would be listed in the meeting agenda. People who contributed to the meeting, as opposed to just listening, rightfully get some reward. 

So I can easily imagine Chantal getting a large percentage of the reward, with smaller amounts apportioned to people who contributed. I don't think it makes sense to reward people who just listened, and it would be an inefficient pain. 

Today things have gotten watered down and I don't think these issues ("[date] RChain Active Members Meeting") deserve a budget of $2000. For instance Rao has stepped up to guide the meetings and various people have played the scribe role, but there's less preparation, no announcements, no report created, and no verbal presentation in the Debrief. Imho the budget should be $800. That's 20 hours of communal work. 

Per meeting the attendees and contributors vary. For this particular meeting (20180314), the attendees were Rao (guide), dckc (scribe), HJ, Rao, JK, Amaka, JW, Karen. Contributions were made by dckc, JK, Amaka, and Karen. I wasn't there so I have nothing to say and shouldn't vote. If @Tonyprisca13 wasn't there then her contribution is small, certainly not 50%. 

If @Tonyprisca13 did some organizing then she would rightfully get a small percentage, and the meeting participants would vote and divide the budget between them, as they see fit. If anybody didn't attend the meeting then they shouldn't be voting. 

ps: @Tonyprisca13, you could possibly take on all of Chantal's role and tasks. A lot of people would welcome having a regular guide who steers the meeting every week. In that case the budget would increase, depending on how many of the tasks get fulfilled.

**Comment by Tonyprisca13** [comment]
@kitblake thank you for posting the antecedent it gives me more insight on what is expected.

**Comment by Tonyprisca13** [comment]
Based on the antecedent:
I prepare the agenda ,
scan issue from github,
participate in the meeting but wasnt the host, 
prepare the Gregs debrief. for 20180314 meeting

**Comment by Tonyprisca13** [comment]
I would like every one who participate in the meeting to vote what they think is justified.

**Comment by jimscarver** [comment]
@dckc you were asked, perhaps with not enough time to answer.Tony did answer so I questioned why you thought he was not acting in good faith. Seemed like Ad Hominem when all that was needed was to vote it down. However this was valuable in clearly defining the tasks involved and norms suggested for distributing it.

**Comment by Tonyprisca13** [comment]
@dckc is voted me a negative of -50 this not a fair judgement

**Comment by Tonyprisca13** [comment]
I am truely this discouraged with this i did more work and i am voted a negative percentage  while somewho did less than 5 percent of the work gets a 20 percent vote

**Comment by Ojimadu** [comment]
IIRC, Tonyprisca did a sizable chunk of work on this issue deserving a greater percentage but was not the host of the meeting. I also don't think the rewards should be split evenly among participants I am more inclined to 1 or 2 percent as persons who hosted, scribed, contributed to the agenda or did more work should get more percentage as without these there would not be a meeting. In previous meetings hosted by chantal on average 50% was voted to her as she mainly did most of the work and the remaining percentage distributed among other participants.

**Comment by kitblake** [comment]
@BelovedAquila @Keaycee @Tonyprisca13 
Please reduce your budgets based on this comment:
https://github.com/rchain/bounties/issues/469#issuecomment-377802761
Adjusting your budget is kinda hidden. You have to click on your budget number (in the Amount column of the table), then click "Open this record in the Budget Vote table". Then at the top will be a Delete button. Click the Delete button and then re-add your new vote.

**Comment by kitblake** [comment]
@Ojimadu Thanks for chiming in. Finding a balance for these meetings is tricky but I think we're working it out for the future. 

However for this 20180314 meeting, things are not right. People have voted a reward for me while I wasn't even there. According to the minutes, @BelovedAquila wasn't there either but she's been assigned a reward too. According to the minutes, @Tonyprisca13 wasn't there either.  

This is what Chantal did:
- [x] set agenda before 2018-01-24
- [x] scan issues in Github and Discord to talk about
- [x] publish meeting announcement in media channels
- [x] participate in the meeting
- [x] make minutes and report
- [x] prepare report for Greg's Debrief 2018-01-31

This is what Tony did:
- prepare the agenda
- scan issue from github
- participate in the meeting but wasn't the host
- prepare Greg's debrief for 20180314 meeting

If Chantal rightfully gets 50%, how much would be fair for @Tonyprisca13 ?
As @dckc said above, I think "perhaps 15 to 20% of the value of the meeting", and that's what her percentage is now.

**Comment by Tonyprisca13** [comment]
@kitblake i was present in the meeting the list on the attendee was based on those who contribute no based on the participants.

**Comment by kitblake** [comment]
In the minutes I see a list of attendees:
Attendees  |  Rao (guide), dckc (scribe), HJ, Rao, JK, Amaka, JW, Karen 

@Tonyprisca13 But if you were attending that doesn't change my thinking about the percentage.

**Comment by Ojimadu** [comment]
Yeah, finding a balance on many issues not just this one is tricky. @kitblake, from the markers you posted above, she did 4/6 of what Chantal does that equates 66.7 percent of the work. If Chantal get 50%, and @Tonyprisca13 did 66.7% of what Chantal would do then 66.7% of would equate 33.35% of the overall work done ((66.7/100)*50/1).

Also, I don't know who wrote the list up there because some names are not there. This was the week US switched to DST so a number of participants missed the meeting but it was more than 8 participants.

**Comment by Tonyprisca13** [comment]
@BelovedAquila made a comment that gave birth to streamline bounty payment which was part of the agenda

**Comment by kitblake** [comment]
@Ojimadu I appreciate your take. I'm still not convinced because I think it's about the (foreground) role that Chantal played, compared to the (background) role that @Tonyprisca13 plays. But I think we've established that 50% is too high, and there's a gray area under 33% where people should vote as they see fit. 

I still think people who attended, but didn't contribute as an agenda item, don't need to be rewarded. What will happen if we have 30 people, and 5 of them presented something? Vote 1% for 25 people? Split the remaining 75% with the contributors and organizer? That's a lot of votes – and time – to have to make.

**Comment by kitblake** [comment]
@Tonyprisca13 Now I see @BelovedAquila's comment. Awarding her 5-10% of the budget for a comment seems excessive to me but that's what you guys voted. 

When I look at the table of RewardVotes for this issue it seems biased. It looks like there are people who make sure that their friends get rewards. Maybe this isn't the case, but it's the perception that matters. 

That's why 'Central Control' had to convene last Saturday and place dissenting votes in various issues. The perception is that the bounty system is being abused and exploited. 

It's important to realize that Central Control didn't exist until last Saturday. It's because of issues like this one that the group gathered and took action. Nobody is happy about it.

**Comment by dckc** [comment]
@Tonyprisca13 would you please tell us more about yourself? What is your background? What skills and interests do you bring to the RChain project? Your github profile says very little about you. I don't see anything about you from the introductions form (#222). I don't see a linkedin profile. You go by different names in github and google docs. It would be easier to trust that you're acting in good faith if it were clear who you are and that you have significant reputation at stake.

About the record of attendees...

> I don't know who wrote the list up there. ...

I did (as anyone who checks the Version history of the document can see). I wanted to count the number of participants in order to refine my "If there are 10 people at the meeting..." comment, so I went through the named speakers.

Attributing comments to speakers is part of the tradition of accountability and quality meetings that I (along with Ralph Swick and others) brought to W3C:

> Meeting time is very valuable; for example, with 12 people in a meeting, five minutes is one person-hour. Good records preserve the value of the time spent in meetings.  -- [MeetingRecords in the W3C wiki](https://www.w3.org/wiki/MeetingRecords)

If you look at notes from [ECMA JavaScript standardization meetings](http://tc39.github.io/tc39-notes/), you'll see the tradition has been picked up there too.

I hold office hours Saturday mornings (#413) to share this sort of project management experience with the RChain community, by the way. And I have contributed to this level of quality records in [executive committee records](https://www.gov.rchain.coop/executive-committee) as well.

I'm interested to know  what "preparing the agenda" consists of. As far as I can tell, it consists of adding 5 bullet points to the March 11, 11:27PM (Chicago time) version of the document. Each had an issue number and one or more names. @Tonyprisca13 , did you confirm that the named people intended to attend the meeting? Did you even notify them that they were expected?

Amaka added several items in the March 12, 11:04 AM version. In the March 12, 12:48 PM version, I sync'd the agenda with the **RAM-weekly-agenda** label.

@Ojimadu writes:
> This was the week US switched to DST so a number of participants missed the meeting

The agenda had inconsistent information on the start time; it gave 20:00 UTC but it said " after Greg's Debrief" which was actually 19:00 UTC. It appears H J introduced this copy-and-paste error, but anyone claiming to have prepared the agenda is responsible to see that such details are right.

I'm also interested to know what "prepare Greg's debrief for 20180314 meeting" refers to. I don't recall anyone but H J or Chantal giving those reports.
